api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -387,30 +406,25 @@ type RuledConfigs struct {
 	Number int `json:"number"`
 }
 
-// LoadConcurrency is the config for data path load concurrency per node.
-type LoadConcurrency struct {
-	// GlobalConfig specifies the concurrency number to all nodes for which per-node config is not specified
-	GlobalConfig int `json:"globalConfig,omitempty"`
-
-	// PerNodeConfig specifies the concurrency number to nodes matched by rules
-	PerNodeConfig []RuledConfigs `json:"perNodeConfig,omitempty"`
-}
-
+// NodeAgentConfigMapSettings is the config for node-agent
+// Struct should be same as:
```

Comment:
would move this comment to another place, to avoid it being propagated to yaml description

Reply:
This is not a problem for the NodeAgentConfigMapSettings, because it's included inline, so this comment is not propagated to yaml, but you are right other similar comments that are not included inline should be moved out. Will change this.